lib/internal/modules/cjs/loader.js

```diff
@@ -1120,10 +1128,14 @@ Module._extensions['.js'] = function(module, filename) {
     const pkg = readPackageScope(filename);
     // Function require shouldn't be used in ES modules.
     if (pkg?.data?.type === 'module') {
-      const parent = moduleParentCache.get(module);
-      const parentPath = parent?.filename;
-      const packageJsonPath = path.resolve(pkg.path, 'package.json');
-      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
+      process.emitWarning('Attempting to require and ESModule.' +
```

Comment:
```suggestion
      process.emitWarning('Attempting to require and ESModule. ' +
```
[Tests](https://github.com/nodejs/node/runs/2851669236?check_suite_focus=true)